Extension/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.14.6: Janurary 2, 2017
+## Version 0.14.6: Janurary 4, 2017
```

Comment:
I believe January 10 is when we're planning to submit this update

Reply:
I thought we were talking about the 16th per Rong's mail?